src/uct/tcp/tcp_iface.c

```diff
@@ -167,8 +171,14 @@ uct_tcp_iface_get_address(uct_iface_h tl_iface, uct_iface_addr_t *addr)
     uct_tcp_iface_t *iface           = ucs_derived_of(tl_iface,
                                                       uct_tcp_iface_t);
     uct_tcp_iface_addr_t *iface_addr = (uct_tcp_iface_addr_t*)addr;
+    ucs_status_t status;
+    uint16_t port;
+
+    status = ucs_sockaddr_get_port((struct sockaddr*)&iface->config.ifaddr,
+                                   &port);
+    ucs_assert_always(status == UCS_OK);
+    iface_addr->port = htons(port);
```

Comment:
add empty line between `ucs_assert_always()` and `iface_addr->port = htons(port);`